docs/orchestrating-elastic-stack-applications/apm-server.asciidoc

```diff
@@ -23,11 +23,11 @@ This section describes how to deploy, configure and access an APM Server with EC
 [id="{p}-apm-eck-managed-es"]
 == Use an Elasticsearch cluster managed by ECK
 
-Managing both APM Server and Elasticsearch by ECK allows a smooth and secured integration between the two. The output configuration of the APM Server is setup automatically to establish a trust relationship with Elasticsearch.
+Managing the APM Server a Kibana instance and Elasticsearch with ECK allows a smooth and secured integration between the stack components. The output configuration of the APM Server is setup automatically to establish a trust relationship with Elasticsearch. Specifying the Kibana reference allows ECK to automatically configure the Kibana endpoint as described in https://www.elastic.co/guide/en/apm/server/current/setup-kibana-endpoint.html
```

Comment:
I find this first sentence a bit hard to parse. Not sure if I have a better suggestion though. 
```suggestion
Managing APM Server, Kibana and Elasticsearch with ECK allows a smooth and secured integration between the stack components. The output configuration of the APM Server is setup automatically to establish a trust relationship with Elasticsearch. Specifying the Kibana reference allows ECK to automatically configure the  link:https://www.elastic.co/guide/en/apm/server/current/setup-kibana-endpoint.html[Kibana endpoint].
```